packages/ui-flex/src/Flex/README.md

```diff
@@ -279,25 +279,28 @@ type: example
 ### Handling overflow
 
 When `direction` is set to `column`, Flex.Items' `overflowY` property is automagically set
-to `auto` to account for content overflow with a vertical scrollbar.
+to `auto` to account for content overflow with a vertical scrollbar. Add padding, so focus rings are not cut off.
```

Comment:
I don't like that we are doing this automagically, but I think it would be a breaking change to remove